composer.json

```diff
@@ -23,7 +23,6 @@
     "require": {
         "php": "^7.2",
         "doctrine/common": "^2.0",
-        "friendsofsymfony/user-bundle": "^2.0",
```

Comment:
@wbloszyk I removed FosUserBundle from other project and I needed to add

- symfony/security-bundle ("maybe" you don't need this because we are adding `symfony/security-core`)
- doctrine/annotations
- symfony/validator